lib/unleash/bootstrap.rb

```diff
@@ -0,0 +1,55 @@
+require 'unleash/configuration'
+require 'unleash/feature_toggle'
+require 'logger'
+require 'time'
+require 'net/http'
+require 'uri'
+
+module Unleash
```

Comment:
It would be nice to also see a `data` bootstrapper out of the box.

Reply:
So, this is now possible after #86 got merged.

The main concern is that the data would need to be serialized in JSON first (to only then be de-serialized).

Having an extra `Unleash::Bootstrap::FromData` class doesn't seem like a bad idea though.